cognite_toolkit/_cdf_tk/utils/auth.py

```diff
@@ -20,32 +21,34 @@
 from cognite_toolkit._cdf_tk.utils import humanize_collection
 from cognite_toolkit._version import __version__
 
+Provider: TypeAlias = Literal["entra_id", "auth0", "other"]
 LoginFlow: TypeAlias = Literal["client_credentials", "token", "device_code", "interactive"]
-Provider: TypeAlias = Literal["entra_id", "other"]
-VALID_LOGIN_FLOWS = get_args(LoginFlow)
 VALID_PROVIDERS = get_args(Provider)
+VALID_LOGIN_FLOWS = get_args(LoginFlow)
 
 CLIENT_NAME = f"CDF-Toolkit:{__version__}"
+PROVIDER_DESCRIPTION = {
+    "entra_id": "Use Microsoft Entra ID to authenticate",
+    "auth0": "Use Auth0 to authenticate",
+    # "cdf": "Use Cognite IDP to authenticate",
+    "other": "Use other IDP to authenticate",
+}
```

Comment:
I switched the order in the code base such that it is 1. provider 2. login flow, to make it consistent with how the user is prompted.